examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>why this step is required? why one needs to prepare model in their local? wont it be confusing?  What's the connection between training model on local and training on AWS?</p><p> </p><p>also the title <code>preparing</code> looks confusing too. isnt it just <code>training and exporting model configs</code> on   local machine?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>This section on local training is not necessary. I will remove the local training section, and simply direct the user to one of the existing example notebooks.</p>

Reply:
<p>I removed the code related to local training and renamed the section.</p>